Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -23,5 +23,9 @@
     <Reference Include="System.Configuration" Condition=" $(Features.Contains('FEATURE_CONFIGURATION')) " />
     <Reference Include="System.Runtime.Remoting" Condition=" $(Features.Contains('FEATURE_REMOTING')) " />
     <Reference Include="System.Xaml" Condition=" $(Features.Contains('FEATURE_WPF')) " />
-   </ItemGroup>
+  </ItemGroup>
+  <Import Project="$(BeforeTargetFiles)" />
+  <Target Name="BeforeBuild" DependsOnTargets="$(BeforeTargets);GenerateCurrentVersion" />
```

Comment:
I think these won't work properly (same issue as before, the target name BeforeBuild doesn't seem to work with the new project files).